.changeset/rare-nails-knock.md

```diff
@@ -0,0 +1,5 @@
+---
+"@alephium/shared": patch
+---
+
+Export NFT schema check and saniizing functions
```

Comment:
Typo.

nit: Also, I don't find this a particularly useful entry in our changelog. I'd replace it with

```suggestion
Sanitize NFT data
```

Reply:
Done